README.md

```diff
@@ -18,3 +18,21 @@ So that we can contribute to high employee delight among our peers through lower
 I want to provide value **using technologies I do not typically have an opportunity to work in**,
 
 So that I can *broaden my technical acumen* on an exercise that *feels more valuable than tutorials*.
+
+# To run this application:
```

Comment:
I like the multi-layer README, where this one is more all encompassing and the one in the source code directory is a bit more technical on individual commands.